app/models/spree/checkout_decorator.rb

```diff
@@ -1,75 +1,67 @@
 module Sprangular::OrderCheckoutDecorator
+  def update_from_params(params, permitted_params, request_env = {})
+    success = false
+    @updating_params = params
+    run_callbacks :updating_from_params do
+      attributes = @updating_params[:order] ? @updating_params[:order].permit(permitted_params).delete_if { |k,v| v.nil? } : {}
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Line is too long. [127/80]<br>Space missing after comma.